web/src/components/Selector.tsx

```diff
@@ -97,14 +97,14 @@ const Selector = ({
             onClick={closeDropDown}
             $enabled
             $selected={item.code === activeItemCode}>
-            <BoldSpacer>{item.name}</BoldSpacer>
+            <BoldSpacer />
             {item.name}
           </Element>
         )
       }
       return (
         <Element as={Tooltip} key={item.code} text={disabledItemTooltip} flow='up' $enabled={false} $selected={false}>
-          <BoldSpacer>{item.name}</BoldSpacer>
+          <BoldSpacer />
```

Comment:
It think you should revert this. 
It was added to fix the jumping of the languages while hovering, because when they are hovered they get bold and therefore jump around.